arch/power/slide_ppc_tpl.h

```diff
@@ -0,0 +1,31 @@
+/* Optimized slide_hash for PowerPC processors
+ * Copyright (C) 2017-2021 Mika T. Lindqvist <postmaster@raasu.org>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include <altivec.h>
```

Comment:
I think using a template should be a separate PR that maybe could include other architectures.

Reply:
I just wanted to include it here, so everything is isolated to PowerPC related code. We can do separate pull request for other architectures if needed.